spec/support/mockers/equipment_model.rb

```diff
@@ -0,0 +1,31 @@
+require Rails.root.join('spec/support/mockers/mocker.rb')
```

Comment:
Why do these need to be required manually? Doesn't RSpec require everything in the `spec/support` directory automatically?